data/yam/autoyast/support_images/sles12sp5_install_default_pattern_aarch64.xml.ep

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why still named as '.ep', I haven't seen the needed for it, please remove the extension name of 'ep' for all  the 4 files.

Reply:
renamed.